crypto/src/constants.rs

```diff
@@ -0,0 +1,5 @@
+pub const MEMO_CIPHERTEXT_LEN_BYTES: usize = 528;
+pub const OVK_WRAPPED_LEN_BYTES: usize = 80;
+
+// This is the `MEMO_CIPHERTEXT_LEN_BYTES` - MAC size (16 bytes).
+pub const MEMO_LEN_BYTES: usize = 512;
```

Comment:
do we want to have a dedicated, catch-all constants module, or should we define constants locally in the modules they relate to? currently we've been doing the latter, should we switch to the former?

Reply:
ah word, I'll remove the constants module and we can still with defining constants in the relevant module